library/src/main/java/org/apache/apex/malhar/lib/window/impl/SpillableSessionWindowedStorage.java

```diff
@@ -52,10 +53,13 @@ public void setup(Context.OperatorContext context)
     if (keyToWindowsMap == null) {
       // NOTE: this will pose difficulties when we try to assign the entries to a time bucket later on.
       // This is logged in APEXMALHAR-2271
-      keyToWindowsMap = scc.newSpillableSetMultimap(bucket, keySerde, (Serde<Window.SessionWindow<K>>)(Serde)windowSerde);
+      // A work around to make session window data never expire and all kept in one time bucket
+      keyToWindowsMap = scc.newSpillableSetMultimap(bucket, keySerde, (Serde<Window.SessionWindow<K>>)(Serde)windowSerde, new TimeExtractor.FixedTimeExtractor(Long.MAX_VALUE));
```

Comment:
That would mean no window is never going to be purged and it will grow indefinitely.